pkg/db/migrations/000011_remove_license_fields.up.sql

```diff
@@ -0,0 +1,83 @@
+-- SPDX-FileCopyrightText: 2025 Siemens AG
+-- SPDX-FileCopyrightText: 2025 Dearsh Oberoi <oberoidearsh@gmail.com>
+-- SPDX-License-Identifier: GPL-2.0-only
+
+ALTER TABLE license_dbs
+DROP COLUMN rf_fsffree,
+DROP COLUMN rf_gplv2compatible,
+DROP COLUMN rf_gplv3compatible,
+DROP COLUMN rf_fedora,
+DROP COLUMN rf_detector_type,
+DROP COLUMN marydone;
```

Comment:
Just a question, Are we sure that there are no other functionality that depends on this table? We should be 100% sure before coming up with DROP Columns.